openedx_ledger/admin.py

```diff
@@ -81,14 +89,33 @@ class Meta:
         model = models.Transaction
         fields = '__all__'
 
-    search_fields = ('content_key', 'lms_user_id', 'uuid', 'external_reference__external_reference_id',)
-    _all_fields = [field.name for field in models.Transaction._meta.get_fields() if field.name != 'external_reference']
-    list_display = ('uuid', 'idempotency_key', 'quantity', 'state',)
+    search_fields = (
+        'content_key',
+        'lms_user_id',
+        'uuid',
+        'external_reference__external_reference_id',
+        'subsidy_access_policy_uuid',
+    )
+    _all_fields = [
+        field.name for field in models.Transaction._meta.get_fields()
+        if field.name != 'external_reference'
+    ]
+    _writable_fields = [
+        'fulfillment_identifier',
+    ]
+    list_display = (
```

Comment:
I wonder if we could also include a boolean value in the `list_display` for whether or not the transaction has an associated `Reversal`?